data/json/npcs/lumbermill_employees/TALK_lumbermill_fabricate.json

```
@@ -177,15 +190,13 @@
   {
     "type": "effect_on_condition",
     "id": "EOC_BUY_BEAM_TRUE_EFFECT",
-    "global": false,
     "effect": [
       { "npc_add_effect": "lumbermill_waiting_for_wood_beam", "duration": "4 h" },
       { "npc_add_effect": "currently_busy", "duration": "4 h" }
     ]
   },
   {
     "type": "effect_on_condition",
-    "id": "EOC_LUMBER_BUY_FALSE_EFFECT",
-    "global": false
+    "id": "EOC_LUMBER_BUY_FALSE_EFFECT"
```

```suggestion
    "id": "EOC_LUMBER_BUY_FALSE_EFFECT",
    "effect": []
```
The string extractor is expecting an "effect" field here, but it can just be blank.